apps/ledger-live-desktop/src/renderer/analytics/mixpanel.ts

```diff
@@ -0,0 +1,12 @@
+import mixpanel from "mixpanel-browser";
+
+const API_HOST = "https://api.mixpanel.com";
+
+const TOKEN = "9a6672bcb9778cb9ea6e6ba671690d66";
```

Comment:
We should never add token like that, it can introduce security issue, 

you need to add it to the project and use `env` var